src/contributing/void-docs/submitting.md

```diff
@@ -0,0 +1,56 @@
+## Submitting Changes
+
+### Requirements
+
+To clone the repository and push changes git is required.
+
+```
+# xbps-install -S git
+```
+
+Building the Void Handbook locally requires
+[mdBook](https://rust-lang-nursery.github.io/mdBook/) to be installed. To get
+it:
+
+```
+# xbps-install -S mdBook
+```
+
+### Forking
+
+To fork the repository a [github account](https://github.com/join) is needed.
+After the account is created follow github's
+[guide](https://help.github.com/en/articles/fork-a-repo) on setting up a fork.
+
+Create a new
+[branch](https://help.github.com/en/articles/creating-and-deleting-branches-within-your-repository)
+to edit, and then clone the repository onto your computer and enter it:
+
+```
+$ git clone https://github.com/YOUR_USERNAME/void-docs.git
+$ cd void-docs
+```
+
+After editing the file(s), commit the changes and push them to the forked
+repository:
+
+```
+$ git add <EDITED_FILE>
+$ git commit -m "<COMMIT_MESSAGE>"
+$ git push
+```
+
+> The commit message should be in the form of `something: reduced section x and
+> moved to it's own page`
+
+Pull requests should only contain a single commit. If a change is made after the
+initial commit, or the commit message needs to be updated the changes will need
```

Comment:
I think it would be clearer without everything between 'initial commit,' and '`git add`' that's probably all we need to get into here.